target/raspberry-pi2_3/profile.env

```diff
@@ -32,7 +32,7 @@ export ANT_DELETESEM_PATH="/usr/bin/ant-deps/deletesem"
 # Runtime directory Path
 export ANT_BIN_DIR="/etc/ant/bin"
 export ANT_CONFIG_DIR="/etc/ant/config"
-export ANT_SENSOR_DRIVER_DIR="/etc/ant/sensor-drivers"
+export ANT_SENSOR_DRIVER_DIR="/etc/ant/device-drivers"
```

Comment:
As now, changing ```ANT_SENSOR_DRIVER_DIR```'s name would be costly.
However, in the long term, it need to be changed.